Google.Api.Gax.Grpc/Rest/PartialDecodingStreamReader.cs

```diff
@@ -83,6 +83,7 @@ internal class PartialDecodingStreamReader<TResponse> : IAsyncStreamReader<TResp
     /// </summary>
     private bool _endOfData;
 
+    // TODO: retain an exception instead of a message?
```

Comment:
Probably an ExceptionDispatchInfo would be better. Fine to do that in the next iteration.

Reply:
Removed this comment entirely, but added more detail to the earlier TODO.